tc/sgx/trusted_worker_manager/enclave_untrusted/enclave_bridge/kme/signup_kme.cpp

```diff
@@ -14,6 +14,8 @@
  */
 
 #include "kme_enclave_u.h"
+#include "sgx_uae_quote_ex.h"
+#include "sgx_uae_epid.h"
```

Comment:
I don't find SGX SDK header files included explicitly in `signup_kme.cpp` earlier ? Is this some new dependency in KME for SDK 2.10 ?

Reply:
the SGX header files were indirectly included from enclave.h (like `sgx_uae_service.h`). I removed these redundant header files which are already included in enclave.h